### DISCUSSION TITLE
Looking for the RexxLexer source

### DISCUSSION THREAD

**Discussion by rvjansen** [pygments/pygments#2562] [Q&A] [answered]
My apologies in advance, but ... I was looking to check the Rexx lexer with minted in xelatex - and would like to have a peek at the source for Rexx to see if it is feasible to morph that into a set of ooRexx and NetRexx lexers. I cannot find Rexx anywhere in the source tree.

I only see Rexx in an examples directory:
```
➜  pygments git:(master) find . | grep rexx
./tests/examplefiles/rexx
./tests/examplefiles/rexx/example.rexx
./tests/examplefiles/rexx/example.rexx.output
➜  pygments git:(master)
```

I am probably missing something ... who can point me into the right direction?

best regards,

René.

**Comment by jeanas** [comment, marked as answer]
`git grep rexx`?

**Reply by rvjansen** [reply]
Check! Thx.
